skbio/stats/composition.py

```diff
@@ -608,10 +610,271 @@ def centralize(mat):
 
     """
     mat = closure(mat)
-    cen = ss.gmean(mat, axis=0)
+    cen = scipy.stats.gmean(mat, axis=0)
     return perturb_inv(mat, cen)
 
 
+@experimental(as_of="0.4.0")
+def ancom(mat, cats,
+          alpha=0.05,
+          multicorr=False,
```

Comment:
Is there a better name for this? My first thought without knowing anything was that `multicorr` was for `multi-correlation` not Bonferroni.